src/clj/rems/db/users.clj

```diff
@@ -69,7 +70,11 @@
        (map :userid)
        (doall)))
 
-(defn get-user [userid]
-  (->> userid
-       get-user-attributes
-       format-user))
+(defn get-user
+  "Given a userid, returns a map with keys :userid, :mail and :name."
```

Comment:
nit: :mail -> :email

Reply:
Hmm, the behavior in the case of a non-existent user is slightly complicated, so it might be good to document it in the docstring. (I see that it is documented in the test case added for this function, but having it in the docstring would make it more explicit.)